src/ext/java/org/opentripplanner/ext/geocoder/StopCluster.java

```diff
@@ -18,10 +19,22 @@ record StopCluster(
   @Nullable String code,
   String name,
   Coordinate coordinate,
-  Collection<String> modes
+  Collection<String> modes,
+  List<Agency> agencies,
```

Comment:
Could you describe a case where multiple agencies would be returned?

Reply:
In the Altanta region, for the stop cluster formed by stops with code 600011 (from 3 different feeds for Cobb, Gwinnett, and Xpress), the logic as written returns one stop as documented, however only one agency is listed. Is that intended?

Reply:
I see now. See my comment in GeocoderAPI.md.